packages/xo-server/src/xo-mixins/proxies.mjs

```diff
@@ -166,13 +177,19 @@ export default class Proxy {
   async getProxy(id) {
     const proxy = await this._getProxy(id)
     addProxyUrl.call(this, proxy)
+    await getProxyVersion.call(this, proxy)
```

Comment:
What do you think about having a single function that calls both of your functions?
In this case, if we need to add more things "on the fly" later, we will have to add the new things only in one place.
E.g.
```js
...

async function populateProxy(proxy){
 addProxyUrl(proxy)
 await addProxyVersion(proxy)
} 
...

export default class Proxy {
   ...
  async updateProxy(id){
   ...
   await populateProxy(proxy)
   return proxy
  }
 
  async getProxy(id){
   ...
    await populateProxy(proxy)
    return proxy
  }
...
}
```